atomicapp/cli/main.py

```diff
@@ -365,6 +375,21 @@ def create_parser(self):
             help='The name of a container image containing an Atomic App.')
         gena_subparser.set_defaults(func=cli_genanswers)
 
+        # === "init" SUBPARSER ===
+        init_subparser = toplevel_subparsers.add_parser(
+            "init", parents=[globals_parser])
```

Comment:
okay, removed parents=[global_parser] and got this:
```
dropbox/dev/atomicapp  pr_603 ✗                                                                                        22m ⚑  
▶ sudo atomicapp init --help
usage: atomicapp init [-h] [--destination DESTINATION] app_name

positional arguments:
  app_name              App name.

optional arguments:
  -h, --help            show this help message and exit
  --destination DESTINATION
                        Path to the directory where the Atomic App is to be
                        initialized.
```

which is much better :+1: